server/src/main/resources/db/V10__create_teams_table.sql

```diff
@@ -1,14 +1,14 @@
-drop table if exists teams;
-CREATE TABLE teams(
+drop table if exists team;
+CREATE TABLE team (
    teamid varchar PRIMARY KEY,
    name varchar UNIQUE,
    description varchar
 );
 
-drop table if exists teamMembers;
-CREATE TABLE teamMembers(
+drop table if exists team_member;
+CREATE TABLE team_member (
```

Comment:
as @mkimberlin said, this needs to be done in an another version. you will have to create another .sql file with V18 or whatever the next available version is and then do something like below. 
```
ALTER TABLE table_name
RENAME TO new_table_name;
```

Reply:
updated